soroban-env-host/src/storage.rs

```diff
@@ -17,6 +17,9 @@ use crate::xdr::{LedgerEntry, LedgerKey, ScHostStorageErrorCode};
 use crate::Host;
 use crate::{host::metered_map::MeteredOrdMap, HostError};
 
+pub type FootprintType = MeteredOrdMap<Rc<LedgerKey>, AccessType, Budget>;
+pub type StorageMapType = MeteredOrdMap<Rc<LedgerKey>, Option<Rc<LedgerEntry>>, Budget>;
```

Comment:
Suggest naming these `FootprintMap and `StorageMap`, since both are maps.

Reply:
Done.